Beschrijver.hs

```diff
@@ -0,0 +1,66 @@
+module Beschrijver ( beschrijfSentence ) where
+
+import Core
+
+beschrijf :: Char -> String
+beschrijf ' ' = "een spatie"
+beschrijf '!' = "een uitroepteken"
+beschrijf '"' = "een aanhalingsteken"
+beschrijf '#' = "een hekje"
+beschrijf '$' = "een dollarteken"
```

Comment:
Wat is dit nou voor een amerikaans imperialisme, wel een dollar maar geen euro?!?!

Ik stel voor:

```haskell
beschrijf "€" = "een euroteken"

Reply:
```/s/"€"/'€'/g```, maar voor de rest heb ik het toegevoegd.